components/worktemplates/components/menu/use_case.tsx

```diff
@@ -0,0 +1,187 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useMemo} from 'react';
+import {useIntl} from 'react-intl';
+import styled from 'styled-components';
+
+const QuickUse = styled.button`
+    position: absolute;
+    top: 7px;
+    right: 7px;
+
+    text-align: center;
+    padding: 4px 10px;
+    border: 0px;
+
+    background: var(--denim-button-bg);
+    border-radius: 4px;
+    font-weight: 600;
+
+    font-size: 11px;
+    line-height: 16px;
+    color: var(--button-color);
+    visibility: hidden;
+    opacity: 0;
+    z-index: 2;
+
+    transition: visibility 0.4s ease-in-out, opacity 0.4s ease-in-out;
+`;
+
+interface UseCaseProps {
+    className?: string;
+    name: string;
+    illustration: string;
+    channelsCount: number;
+    boardsCount: number;
+    playbooksCount: number;
+
+    onQuickUse: () => void;
+    onSelectTemplate: () => void;
+}
+
+const UseCase = (props: UseCaseProps) => {
+    const {formatMessage, formatList} = useIntl();
+
+    const details = useMemo(() => {
+        const detailBuilder: string[] = [];
+        if (props.channelsCount) {
+            detailBuilder.push(formatMessage({
+                id: 'worktemplates.menu.usecase_channels_count',
+                defaultMessage: '{channelsCount, plural, =1 {# channel} other {# channels}}',
```

Comment:
I know I commented on using plurals earlier in this PR, but I didn't realize that you could add `#` into them to add the actual count. That's so much easier than adding it separately

Reply:
I could take credit but GitHub Autopilot suggested this for me :P